docs/examples/kitchen-sink/graphviz.rst

```diff
@@ -0,0 +1,28 @@
+========
+Graphviz
+========
+
+Inheritance Diagram
+-------------------
+
+Using `sphinx.ext.inheritance_diagram`, inheritance diagrams can be generated
+through `sphinx.ext.graphviz`.  If the output of the inheritance diagrams are
```

Comment:
we don't have intersphinx mappings set up (haven't needed them so far) so these won't be links (will get rendered as italics). Can you add intersphinx too?  LMK if you need guidance (or if you prefer I can push the changes to your branch myself).

Reply:
My rst/sphinx game is weak, but I figured it out.  Looks much better!